### PR TITLE
feat: [IXSPD1-1633] update css for tab in security tokens page

### PR DESCRIPTION
**Pull request by galvin96**
## Description

Please describe the purpose of this pull request.

## Changes

- Styles of security tokens/payout events page. Result: 
image

## Attached Links

1. Jira ticket: [IXSPD1-1633]
2. Documents:

## Checklist

- [x] Manual tests
- [x] The pull request doesn't affect existing feature

## For reviewers

- Ensure new code doesn't break existing features
- Reviewer shouldn't merge if there are pending unresolved comments

[IXSPD1-1633]: https://investax.atlassian.net/browse/IXSPD1-1633?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/pages/SecurityTokens/index.tsx [hunk 1]
```diff
@@ -102,12 +102,25 @@ const TabsContainer = styled.div`
...
+  margin: 0 -40px 0 -40px;
+  padding: 0 40px 0 40px;
 `
 
 const TabLabel = styled.div`
-  @media (max-width: ${MEDIA_WIDTHS.upToExtraSmall}px) {
+  /* @media (max-width: ${MEDIA_WIDTHS.upToExtraSmall}px) {
     font-size: 20px;
-  }
+  } */
```
Comment (thi-investax) [inline src/pages/SecurityTokens/index.tsx:114]: why we commented this code @galvin96 ?

Comment (galvin96) [reply src/pages/SecurityTokens/index.tsx:114]: Well, from figma, in mobile layout, we use size 13px for the label, 20px is bigger in this case.

Comment (thi-investax) [reply src/pages/SecurityTokens/index.tsx:114]: image
the text doesn't look good on mobile. Could you check @galvin96 ?

Comment (galvin96) [reply src/pages/SecurityTokens/index.tsx:114]: You should refresh after changing to the mobile layout to see the visual of mobile: 
Screenshot 2024-07-29 at 11 03 09 AM

Comment (thi-investax) [reply src/pages/SecurityTokens/index.tsx]: ok. looks good thanks

**Pull request merged**